testData/sample_tests/test/simple_test.dart

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2019 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+import'package:test/test.dart';
+
+/// Test data for test line marker tests.
+/// The outline for this file is pretty-printed in simple_outline.txt.
+///
+/// To update the outline, open this file in a debugging IntelliJ instance, then
+/// set a breakpoint in the ActiveEditorsOutlineService's OutlineListener call
+/// to `notifyOutlineUpdated`, then look at the Flutter Outline for this file.
+/// This will break at the outline and allow you to copy its json value.
+///
+/// You can pretty-print the json using the pretty_printer.dart script.
```

Comment:
provide a full path to where to find the pretty_printer.dart script.

Reply:
Done.